templates/docs/examples/patterns/contextual-menu/_script.js

```diff
@@ -15,6 +15,15 @@ function toggleMenu(element, show, top) {
     if (typeof top !== 'undefined') {
       target.style.top = top + 'px';
     }
+
+    if (show) {
+      var focusableElements = 'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])';
+      var focusable = target.querySelectorAll(focusableElements);
```

Comment:
Would `querySelector` just do the same and automatically select first element, so we don't need to get it via index?

Reply:
Oh good point, you're right